PingID SDK/create-authz-token

```diff
@@ -52,6 +51,13 @@ var path = path.replace("{{id}}", pm.environment.get("id"));
 var parameters = pm.request.url.query.toString();
 
 // Hash and Hex Encode Request Payload
```

Comment:
this comment should be **just before** line 61

Reply:
Yeah, that makes more sense... I'll move the comment.